scripts/tests/extension-version.test.js

```diff
@@ -0,0 +1,73 @@
+const { expect } = require('chai')
```

Comment:
Do you think that these tests now cover all workflows?

Reply:
I'll read through them a couple more times to be 100% sure about this 👍

Reply:
will merge now 👍